var/spack/repos/builtin/packages/alpaka/package.py

```diff
@@ -33,45 +56,47 @@ class Alpaka(CMakePackage, CudaPackage):
         values=(
             "serial",
             "threads",
-            "fiber",
             "tbb",
             "omp2_gridblock",
             "omp2_blockthread",
-            "omp5",
-            "oacc",
             "cuda",
             "cuda_only",
             "hip",
             "hip_only",
+            "sycl_cpu",
+            "sycl_gpu",
+            "sycl_fpga",
         ),
         description="Backends to enable",
         default="serial",
     )
 
     variant("examples", default=False, description="Build alpaka examples")
 
-    depends_on("boost")
-    depends_on("boost+fiber", when="backend=fiber")
-    depends_on("cmake@3.18:", when="@0.7.0:")
```

Comment:
Since the older versions are deprecated and still included, the `cmake` dependency needs to remain in place as well. Right now, 0.7.0 would not depend on the right minimum version of `cmake`.

Reply:
Added depends for the deprecated versions as well